src/qmflows/components/reactivity.py

```diff
@@ -25,7 +49,14 @@ def get_current_value(self, mol):
         xs = [i - 1 for i in self.atoms]
         return self.fun(conf, *xs)
 
+    @overload
```

Comment:
What is the difference between this overload and using Optional for `value` and `mol`?

Reply:
Overload implies here that there are two different signature:
1. Either ``mol`` is ``None`` and ``value`` has a value.
2. The other way around.

Using ``Optional`` would apply that ``mol`` can be ``None``, ``value`` can be ``None`` or ``mol`` and ``value`` can both be ``None``.